tasks/58_missed-letters/missed-letters.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * findLetter - Find the missing letter in the passed
+ * letter range and return it.
+ *
+ * @example
+ * //return 'd'
+ * findLetter("abce");
+ * @param {String} str - letter range.
+ * @return {String} missing letter.
+ */
+export function findLetter(str) {
+  let lastIndex = str.charCodeAt(0);
+  for (let i = 1; i < str.length; i++) {
+    const currentIndex = str.charCodeAt(i);
+    if (currentIndex - lastIndex !== 1) {
+      return String.fromCharCode(currentIndex - 1);
+    }
+    lastIndex = currentIndex;
+  }
+  return undefined;
```

Comment:
I'd suggest to return empty string here since you defined it in JSDoc

Reply:
I've rewrite JSDoc according out task